docker-compose.evm.yml

```diff
@@ -0,0 +1,22 @@
+version: "2.4"
+services:
+  record-streams-uploader:
+    restart: "no"
+    entrypoint: ["echo", "This container is intentionally disabled by the EVM profile."]
+    command: []
+  account-balances-uploader:
+    restart: "no"
+    entrypoint: [ "echo", "This container is intentionally disabled by the EVM profile." ]
+    command: []
+  record-sidecar-uploader:
+    restart: "no"
+    entrypoint: [ "echo", "This container is intentionally disabled by the EVM profile." ]
+    command: []
+  minio:
+    restart: "no"
+    entrypoint: [ "echo", "This container is intentionally disabled by the EVM profile." ]
+    command: []
+  importer:
+    volumes:
+      - "${NETWORK_NODE_LOGS_ROOT_PATH}/accountBalances/balance0.0.3:/node/streams/accountBalances/balance0.0.3"
+      - "${NETWORK_NODE_LOGS_ROOT_PATH}/recordStreams/record0.0.3:/node/streams/recordstreams/record0.0.3"
```

Comment:
Nodes write to `recordStreams` and uploader converts that to `recordstreams`? That seems a bit silly.

Reply:
Correct, all of our nodes use the case sensitive `recordStreams` folder in actual production environments while the AWS/GCP buckets use the `recordstreams` folder in the bucket path. While I agree it is silly, it has been an issue for quite a while due to discrepancies originating from the 2018/2019 timeframes.

Reply:
This mirrors the current production and pre-production deployments.